src/containers/EventForm/components/ArtworkForm/function.js

```diff
@@ -0,0 +1,28 @@
+import { ArtworksRepository, PeopleRepository } from 'hackoss';
+import { FirebaseApp } from 'utils/firebase';
+
+async function getPeopleBase() {
+  const pplRepo = new PeopleRepository(FirebaseApp);
+  const people = await pplRepo.getPeople();
+  return people;
+}
+export async function getPeople() {
+  const people = await getPeopleBase();
+  return people.map(person => person.name);
+}
+
+export const createArtwork = setStatus => async input => {
+  const pplRepo = new PeopleRepository(FirebaseApp);
+  const awRepo = new ArtworksRepository(FirebaseApp, pplRepo);
+  const exist = await awRepo.getArtworks();
+  if (exist.find(aw => aw.title === input.title)) {
```

Comment:
I was worried we would have different assumptions for the constraints, hence I made ntuoss/hackoss#7 to make sure we're all in the same page. In this case, I would argue that we can have duplicate titles since we're already using IDs to uniquely identify them. Would likely be a similar argument when discussing if workshops can share the same name.

You can check the `constraints` file which defines the constraints.

Reply:
For submitting part I am only checking for duplicates. all others will depend on library constraints.

Reply:
I see! Actually, my point here was we should allow duplicate titles for artwork. But of course this is open for discussion 👯‍♂️

Reply:
I will be resolved when i implement on the part that users choose pictures rather than titles.
will be in TODO.